Sources/KIF/Additions/UIView-Debugging.m

```diff
@@ -8,254 +8,237 @@
 #import "UIView-Debugging.h"
 #import "KIFEventVisualizer.h"
 #import "KIFTouchVisualizerView.h"
+#import <Foundation/Foundation.h>
 
 @implementation UIView (Debugging)
 
 +(void)printViewHierarchy {
+    printf("%s", [self viewHierarchyDescription].UTF8String);
+}
+
++ (NSString *)viewHierarchyDescription {
```

Comment:
almost all of the changes in this file are to update the print methods to support _returning_ an `NSString` instead of printing directly via `printf`